osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -2,154 +2,181 @@
+                Origin = Anchor.Centre,
+            }
+          }
+      };
+    }
 
-        protected override void LoadComplete()
-        {
-            base.LoadComplete();
-            initializeBreaks();
-        }
+    [BackgroundDependencyLoader]
+    private void load(OsuConfigManager config)
+    {
+      // We need BackgroundOpacityOffset to lighten during breaks.
+      if (BackgroundOpacityOffset == null) return;
+
+      lightenDuringBreaks = config.GetBindable<bool>(OsuSetting.LightenDuringBreaks);
+      lightenDuringBreaks.ValueChanged += _ => currentlyOnBreak.TriggerChange();
+
+      currentlyOnBreak.ValueChanged += value => this.TransformBindableTo(
+        BackgroundOpacityOffset,
+        lightenDuringBreaks.Value && value ? 1 : 0,
+        fade_duration,
+        Easing.OutQuint
+      );
+    }
 
-        private void initializeBreaks()
-        {
-            if (!IsLoaded) return; // we need a clock.
+    protected override void LoadComplete()
+    {
+      base.LoadComplete();
+      initializeBreaks();
+    }
 
-            FinishTransforms(true);
-            Scheduler.CancelDelayedTasks();
+    private void initializeBreaks()
+    {
+      if (!IsLoaded) return; // we need a clock.
 
-            if (breaks == null) return; //we need breaks.
+      FinishTransforms(true);
+      Scheduler.CancelDelayedTasks();
 
-            foreach (var b in breaks)
-            {
-                if (!b.HasEffect)
-                    continue;
+      if (breaks == null) return; //we need breaks.
 
-                using (BeginAbsoluteSequence(b.StartTime, true))
-                {
-                    fadeContainer.FadeIn(fade_duration);
-                    breakArrows.Show(fade_duration);
-
-                    remainingTimeAdjustmentBox
-                        .ResizeWidthTo(remaining_time_container_max_size, fade_duration, Easing.OutQuint)
-                        .Delay(b.Duration - fade_duration)
-                        .ResizeWidthTo(0);
-
-                    remainingTimeBox
-                        .ResizeWidthTo(0, b.Duration - fade_duration)
-                        .Then()
-                        .ResizeWidthTo(1);
-
-                    remainingTimeCounter.CountTo(b.Duration).CountTo(0, b.Duration);
-
-                    using (BeginDelayedSequence(b.Duration - fade_duration, true))
-                    {
-                        fadeContainer.FadeOut(fade_duration);
-                        breakArrows.Hide(fade_duration);
-                    }
-                }
-            }
-        }
+      foreach (var b in breaks)
+      {
+        if (!b.HasEffect)
+          continue;
 
-        private void bindProcessor(ScoreProcessor processor)
+        using (BeginAbsoluteSequence(b.StartTime, true))
         {
-            info.AccuracyDisplay.Current.BindTo(processor.Accuracy);
-            info.GradeDisplay.Current.BindTo(processor.Rank);
+          this.TransformBindableTo(currentlyOnBreak, true);
```

Comment:
This can also just be `currentlyOnBreak.Value = true;`. Same way when you change back.

Reply:
That doesn't seem to work

Reply:
Oh. Alright. My bad.